### PR TITLE
One Hot 2d Matrix to Make Sparse 3d Matrix

### PR DESCRIPTION
**Pull request by LeavesBreathe**
for text processing in sequence to sequence

### CODE REVIEW

File: keras/preprocessing/sequence.py [hunk 1]
```diff
@@ -120,3 +120,22 @@ def skipgrams(sequence, vocabulary_size,
...
+'''
+input: 2d matrix with integers
+output: 3d matrix with one hot matrix
+
+intended to be used after 'pad_sequences' has been called, 2d must only have integers
+Assumes 2d shape to be (nb_samples, timesteps)'''
+
+
+def one_hot_2d_matrix(matrix2d):
+	maxinteger = matrix2d.max()
```
Comment (fchollet) [inline keras/preprocessing/sequence.py]: No check that `matrix2d` is a Numpy array...

File: keras/preprocessing/sequence.py [hunk 2]
```diff
@@ -120,3 +120,22 @@ def skipgrams(sequence, vocabulary_size,
...
+	nb_samples, timesteps = matrix2d.shape
```
Comment (fchollet) [inline keras/preprocessing/sequence.py]: No check that `len(shape) == 2`...

File: keras/preprocessing/sequence.py [hunk 3]
```diff
@@ -120,3 +120,22 @@ def skipgrams(sequence, vocabulary_size,
...
+	new_3d_matrix = np.zeros((nb_samples, timesteps, maxinteger+1),dtype=np.bool)
```
Comment (fchollet) [inline keras/preprocessing/sequence.py]: Not style (PEP8) compliant...

**Comment by LeavesBreathe** [comment]
I apologize for the non-checks. Let me close this PR, and if I get around to ensuring it goes through those checks, I will submit another PR! Appreciate the feedback.

**Pull request closed without merging**